src/tool_operate.c

```diff
@@ -2326,6 +2326,16 @@ static CURLcode transfer_per_config(struct GlobalConfig *global,
       else {
         result = FindWin32CACert(config, tls_backend_info->backend,
                                  "curl-ca-bundle.crt");
+#if defined(USE_WIN32_CRYPTO) && \
+    !defined(CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE) && \
+    !defined(LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE)
```

Comment:
Why are these names different
CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE
LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE

How about remove AUTOMATIC_
CURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE
LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE

Reply:
In my idea:
- CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE is only for the command line curl utility. Without this symbol, if curl has no file curl-ca-bundle.crt .

- LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE is for lib, and mean my modification is not included . Pehaps this is not useful : if symbol CURL_WINDOWS_CA_STORE is no set, the modification is just no used.

Reply:
what do you think?

Reply:
This needs a second review I will look it over again soon. I think perhaps it should have a CURLSSLOPT flag instead of set cacert.

Reply:
I like it using this option since it is mutually exclusive, not in addition to.